### ISSUE TITLE
feature request: include ngx_brotli

### ISSUE THREAD

**Issue by rcanavan** [openresty/openresty-packaging#45]
We would like to switch to the official openresty packages instead of compiling it ourselves. However, we would like to use brotli content-encoding for performance reasons. So far, I've not succeeded in building the brotli module for use with the pre-built openresty Alpine packages.  Would it be possible to include mod_brotli in future releases of openresty?

https://github.com/google/ngx_brotli

**Comment by agentzh** [comment]
Yeah we can consider that. Maybe as a dynamic nginx module. Pull requests welcome!
